quil-rs/src/stats/mod.rs

```diff
@@ -0,0 +1,270 @@
+mod execution_graph;
+
+#[cfg(test)]
+pub(crate) mod test_programs;
+
+// Use quil_rs::program::graph for pulse-level programs, but not qubit programs.
```

Comment:
```suggestion
// NOTE: Use quil_rs::program::graph for pulse-level programs, but not qubit programs.
```

Reply:
(Or just delete, but this seems useful to record somewhere)

Reply:
I don't actually recall what this means. It doesn't look like we use `program::graph` anywhere in the `stats` module.